### DISCUSSION TITLE
RouteManager Usage Tips

### DISCUSSION THREAD

**Discussion by Narflar** [Erabior/RouteManager#42] [General]
As this is all new, i'm sure a lot of this will change as the game and mod develop.  But here are a few things I've found to make using RouteManager as reliable as possible.   

- RouteManager does not resume when loading a save!  
  - Remember to check your passenger train when you load up your game, especially if you stopped after you saved!
- Send your train in the right direction!
  - Select only one station at first.  Once going, select the rest.  
    - This forces the train in the direction you want it to go instead of sometimes picking the wrong way.
- Leave the coach cars alone!
  - RouteManager will select the correct stations for your passenger cars.

This is all I can think of that has helped me use the mod.  Please add your tips or tricks if you have any!

**Comment by Acee83** [comment]
Select the stations in one direction first, once the train departs at the first station add the remaining stations to get it running in the direction you want.
Say you start at Bryson and want the train to do Sylva - Alarka, but go to Alarka first:
Only select the stations for Bryson - Alarka at first.
Once the train is leaving Bryson select the remaining stations.
